commit ea0d882fa8c226ac27be4651f9550aa17d75462a
Author: Steve Dickson <steved@redhat.com>
Date:   Mon Nov 21 14:07:58 2022 -0500

    nfs4_getfacl: Initialize acl pointer to NULL
    
    nfs4_getfacl.c: scope_hint: In function 'print_acl_from_path'
    nfs4_getfacl.c:168:17: warning[-Wmaybe-uninitialized]:
        'acl' may be used uninitialized in this function
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/nfs4_getfacl/nfs4_getfacl.c b/nfs4_getfacl/nfs4_getfacl.c
index 954cf7e..ddb3005 100644
--- a/nfs4_getfacl/nfs4_getfacl.c
+++ b/nfs4_getfacl/nfs4_getfacl.c
@@ -148,7 +148,7 @@ out:
 
 static void print_acl_from_path(const char *fpath, enum acl_type type)
 {
-	struct nfs4_acl *acl;
+	struct nfs4_acl *acl = NULL;
 
 	switch (type) {
 	case ACL_TYPE_ACL:
